aries-site/src/data/structures/components.js

```diff
@@ -414,14 +441,19 @@ export const components = [
       'drop down',
       'option list',
       'multi select',
-      'multiple values',
+      'multiple values'
+      'multiple options'
+      'search in dropdown'
+      'search in multiselect'
+      'help text'
+      'scrolling in dropdown',
     ],
   },
   {
     name: 'CheckBox',
     category: 'Inputs',
     description:
-      'When the user needs to select one or more options, use a checkbox.',
+      'CheckBox is a square design element that when clicked accepts the choice(s) offered. Checkboxes allow for multiple selections.',
```

Comment:
```suggestion
      'CheckBox is a square design element that when clicked accepts the choice offered.',
```
CheckBoxGroup would be for multiple choices or options.